app/marketing/management/commands/expiration_start_work.py

```diff
@@ -60,26 +60,49 @@ def handle(self, *args, **options):
         for day in days:
             start_date = (timezone.now() - timezone.timedelta(days=(day+1)))
             end_date = (timezone.now() - timezone.timedelta(days=day))
-            interests_by_created_on = Interest.objects.select_related('profile').filter(
-                created__gte=start_date,
-                created__lt=end_date,
-                pending=False,
-            )
-            interests_by_accepted_on = Interest.objects.select_related('profile').filter(
+
+            profiles = Interest.objects.select_related('profile')
+
+            interests_by_accepted_on = profiles.filter(
```

Comment:
Any thoughts on consolidating these into one `.filter()` via `Q()` as specified in my previous review comments?

Reply:
I'm not terribly against it, but I think that this is a bit clearer -- especially for a first-time contributor who may be unfamiliar with Django -- than `Q()` objects. I can change if you'd rather save the lines of code.